src/Microsoft.Diagnostics.Monitoring.RestServer/CorsConfiguration.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Microsoft.Diagnostics.Monitoring.RestServer
+{
+    public class CorsConfiguration
+    {
+        public string AllowedOrigins { get; set; }
```

Comment:
We don't automatically generate anything in this configuration, correct?

If this list were to include "localhost" it would provide an escalation of privileges attack. I'm fine with handing the user the rope to hang themselves with because they may have a specific scenario involving a localhost allowance but we should make sure we never do it.

Reply:
Correct, the configuration must be explicitly specified by the environment running dotnet-monitor.